config/crd/bases/apps.foundationdb.org_foundationdbclusters.yaml

```diff
@@ -7923,12 +7923,19 @@ spec:
                   type: object
                 routing:
                   properties:
+                    dnsDomain:
+                      maxLength: 253
+                      minLength: 1
+                      pattern: '...'
```

Comment:
Is that pattern here expected?

Reply:
No, this was a missing change from the last commit.